p4-16/psa/PSA.mdk

```diff
@@ -225,6 +225,11 @@ after the Ingress control block has completed executing, based upon
 the contents of several metadata fields in the struct
 `psa_ingress_output_metadata_t`.
 
+Note that if at the end of Ingress control block the egress_port is
```

Comment:
I don't see any way for a PSA implementation to detect that the egress_port "is not set".  How are you thinking it could do that?